azkaban-common/src/main/java/azkaban/metrics/MetricsManager.java

```diff
@@ -74,51 +88,21 @@ public synchronized void startReporting(final String reporterName, final Props p
     final String metricsServerURL = props.get(METRICS_SERVER_URL);
     if (metricsReporterClassName != null && metricsServerURL != null) {
       try {
-        logger.info("metricsReporterClassName: " + metricsReporterClassName);
+        log.info("metricsReporterClassName: " + metricsReporterClassName);
         final Class metricsClass = Class.forName(metricsReporterClassName);
 
-        final Constructor[] constructors =
-            metricsClass.getConstructors();
+        final Constructor[] constructors = metricsClass.getConstructors();
         constructors[0].newInstance(reporterName, this.registry, metricsServerURL);
 
       } catch (final Exception e) {
-        logger.error("Encountered error while loading and instantiating "
+        log.error("Encountered error while loading and instantiating "
             + metricsReporterClassName, e);
-        throw new IllegalStateException(
-            "Encountered error while loading and instantiating "
+        throw new IllegalStateException("Encountered error while loading and instantiating "
                 + metricsReporterClassName, e);
       }
     } else {
-      logger.error("No value for property: "
-          + CUSTOM_METRICS_REPORTER_CLASS_NAME
-          + "or" + METRICS_SERVER_URL + " was found");
-    }
-
-  }
-
-  /**
-   * Create a ConsoleReporter to the AZ Metrics registry.
-   *
-   * @param reportInterval time to wait between dumping metrics to the console
-   */
-  public synchronized void addConsoleReporter(final Duration reportInterval) {
```

Comment:
In such cases, probably we should expose a config parameter that can enable it. Since we are not using it I am thinking of removing it. let me know if you want to restore it. 

We can also probably add that as a reporting feature separately.